workflows/ascc.nf

```diff
@@ -203,83 +240,249 @@ workflow ASCC {
+    //
+    //qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore staxids sscinames sskingdoms sphylums salltitles
+    if ( include_workflow_steps.contains('uniprot_diamond') || include_workflow_steps.contains('ALL') ) {
+        UNIPROT_DIAMOND (
+            modified_input,
+            YAML_INPUT.out.diamond_uniprot_database_path
+        )
+        un_full         = UNIPROT_DIAMOND.out.reformed.map{it[1]}
+        un_hits         = UNIPROT_DIAMOND.out.hits_file.map{it[1]}
+        ch_versions     = ch_versions.mix(UNIPROT_DIAMOND.out.versions)
+    } else {
+        un_hits         = []
+        un_full         = []
+    }
+
+    ch_got_genome = GENERATE_GENOME.out.dot_genome.map{it[1]}
+
+    CREATE_BTK_DATASET (
+        GENERATE_GENOME.out.reference_tuple.first(),
+        ch_got_genome,
+        ch_kmers,
+        ch_tiara.first(),
+        ch_nt_blast,
+        ch_fcsgx.first(),
+        ch_bam.first(),
+        ch_coverage.first(),
+        ch_kraken1.first(),
+        ch_kraken2.first(),
+        ch_kraken3.first(),
+        nt_hits.first(),
+        un_hits.first(),
+        YAML_INPUT.out.ncbi_taxonomy_path.first()
+    )
+    //ch_versions                 = ch_versions.mix(CREATE_BTK_DATASET.out.versions)
+
+
+    //
+    // MODULE: AUTOFILTER ASSEMBLY BY TIARA AND FCSGX RESULTS
+    //
+    if ( include_workflow_steps.contains('tiara') && include_workflow_steps.contains('fcsgx') && include_workflow_steps.contains("autofilter") || include_workflow_steps.contains('ALL') ) {
+        AUTOFILTER_AND_CHECK_ASSEMBLY (
+            YAML_INPUT.out.reference_tuple,
+            EXTRACT_TIARA_HITS.out.ch_tiara,
+            RUN_FCSGX.out.fcsgxresult,
+            YAML_INPUT.out.ncbi_rankedlineage_path
+        )
+        ch_autofiltered_assembly = AUTOFILTER_AND_CHECK_ASSEMBLY.out.decontaminated_assembly.map{it[1]}
+
+        AUTOFILTER_AND_CHECK_ASSEMBLY.out.alarm_file
+            .map { file -> file.text.trim() }
+            .branch { it ->
+                run_btk: "ABNORMAL" ? it.contains("YES_ABNORMAL"): false
+                dont_run: []
+            }
+            .set { btk_bool }
+
+
+        ch_versions              = ch_versions.mix(AUTOFILTER_AND_CHECK_ASSEMBLY.out.versions)
+    } else {
+        ch_autofiltered_assembly = []
+    }
+
+    //
+    // PIPELINE: PREPARE THE DATA FOR USE IN THE SANGER-TOL/BLOBTOOLKIT PIPELINE
+    //              WE ARE USING THE PIPELINE HERE AS A MODULE THIS REQUIRES IT
+    //              TO BE USED AS A AN INTERACTIVE JOB ON WHAT EVER EXECUTOR YOU ARE USING.
+    //              This will also eventually check for the above run_btk boolean from autofilter
+    if ( !exclude_workflow_steps.contains("btk") && include_workflow_steps.contains('busco_btk') && include_workflow_steps.contains("autofilter") && btk_bool.run_btk == "ABNORMAL" || !exclude_workflow_steps.contains("btk") && include_workflow_steps.contains('ALL') ) {
```

Comment:
It's unclear to me why the same thing appears to be called `btk` in `exclude_workflow_steps` and `busco_btk` in `include_workflow_steps`.
Perhaps in this line ` if ( !exclude_workflow_steps.contains("btk") && include_workflow_steps.contains('busco_btk') &&`  instead of `btk` and `busco_btk` it should say `btk_busco`?
`btk_busco` is what the BUSCO-based BlobToolKit pipeline is called in the older version of ASCC.
This is to differentiate it from the parts of ASCC that create a BlobToolKit dataset from the output of ASCC's own processes

Reply:
This was for testing.
Renamed to busco_btk.
Done